doc/api/http.md

```diff
@@ -822,6 +838,8 @@ req.once('response', (res) => {
 });
 ```
 
+Can be passed an instance of {stream.Duplex}, a superclass of Socket.
```

Comment:
Shouldn't the type of the property be changed to `{stream.Duplex}`?

Also, "Can be passed" doesn't sound right for a property that is not a function.